employees/views.py

```diff
@@ -131,7 +131,10 @@ def get_context_data(self, **kwargs: Any) -> dict:
 
     def redirect_to_another_month(self, request: HttpRequest) -> HttpResponseRedirectBase:
         post_data = request.POST.copy()
-        post_data["date"] = datetime.datetime.strptime(post_data["date"], "%m-%Y")
+        try:
+            post_data["date"] = datetime.datetime.strptime(post_data["date"], "%m-%Y")
```

Comment:
@rwrzesien 
one picture worth thousand words ;)
if we manually modify the date in navigation bar
![before](https://user-images.githubusercontent.com/36961162/62393749-5729dc00-b56b-11e9-8a01-5a46fff16bed.png)
we will get an exception
![after](https://user-images.githubusercontent.com/36961162/62393786-6d379c80-b56b-11e9-99f8-315074c851eb.png)
as the input is passed to `datetime.strptime()` function directly. So, when it cannot create valid object, unhanded exception is raised, resulting in `HTTP500`

So I guess we need it. Theoretically, we could use some js script to have a mask here, but this seems just a quicker solution. If you prefer js mask, let me know, I will change.
@kbeker what would be your opinion?

Reply:
Makes sense now, I just thought our code generated those urls which caused errors. But that were those bloody users again :)

Solution is fine, just FYI another way to achieve this would be to define regex in urls.py in a better way that it would catch only proper year range. Then in the case above it would return 404. But as this applies to multiple urls it could be more difficult to maintain.